zuul.d/molecule.yaml

```diff
@@ -1,305 +1,354 @@
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/pkg_build/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-pkg_build
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: pkg_build
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/pkg_build/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/polarion/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-polarion
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: polarion
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/polarion/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/registry_deploy/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-registry_deploy
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: registry_deploy
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/registry_deploy/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/repo_setup/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-repo_setup
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: repo_setup
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/repo_setup/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/rhol_crc/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-rhol_crc
     nodeset: centos-9-crc-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: rhol_crc
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/rhol_crc/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/run_hook/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-run_hook
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: run_hook
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/run_hook/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/set_openstack_containers/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-set_openstack_containers
-    parent: cifmw-molecule-base-crc
```

Comment:
We need to keep this parent to avoid to molecule job failure.

Reply:
keeping